notebooks/Advanced-FuncXClient.ipynb

```diff
@@ -146,18 +151,19 @@
    "source": [
     "items = [1, 2, 3, 4, 5]\n",
     "\n",
-    "res = fxc.run(items, endpoint_id=tutorial_endpoint, function_id=sum_function)\n",
+    "res = gcc.run(items, endpoint_id=tutorial_endpoint, function_id=sum_function)\n",
     "\n",
-    "print (fxc.get_result(res))"
+    "print (gcc.get_result(res))"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Functions with dependencies\n",
     "\n",
-    "funcX requires that functions explictly state all dependencies within the function body. It also assumes that the dependent libraries are available on the endpoint in which the function will execute.  For example, in the following function, we explictly import the time module. "
+    "Globus Compute requires that functions explictly state all dependencies within the function body. It also assumes that the dependent libraries are available on the endpoint in which the function will execute.  For example, in the following function, we explictly import the time module. "
```

Comment:
If you were to make any changes, might as well remove the extra space at the end.

Reply:
Should be good now.